utils/gen-automata/http.py

```diff
@@ -322,23 +322,46 @@ def create_folder_header():
 
 
 def auth_value(name):
+    name_unquoted = f"{name}Unquoted" if name != None else None
+
     auto = Automaton()
     start = auto.start()
-    value = auto.add_state()
+    quote = auto.add_state()
+    value_quoted = auto.add_state()
+    value_quoted = auto.add_state()
```

Comment:
Do we need two value_quoted?

Reply:
Oh, that's a typo I missed. I removed it.